package.json

```diff
@@ -79,13 +19,14 @@
     "lodash": "^3.10.1",
     "marked": "^0.3.6",
     "popper.js": "^1.12.5",
-    "prop-types": "^15.5.8",
+    "prettier": "^1.11.1",
```

Comment:
Prettier should be in `devDependencies` not `dependencies`

Reply:
Fixed!